zerver/openapi/markdown_extension.py

```diff
@@ -253,10 +275,10 @@ def generate_curl_example(
     api_url: str,
     auth_email: str = DEFAULT_AUTH_EMAIL,
     auth_api_key: str = DEFAULT_AUTH_API_KEY,
-    exclude: Optional[List[str]] = None,
-    include: Optional[List[str]] = None,
+    exclude: List[str] = list(),
+    include: List[str] = list(),
```

Comment:
I think the model wit these parameters being optional (with None / not sent meaning that the behavior was not requested) makes more sense to me.

Reply:
Previously, the "include nothing" was modelled as `[""]`, which is still valid(and hence no diffs here). I had to remove None because I'm combining ones in the dict with the harcoded ones, and None gives problems there.